app/database/manager/index.ts

```diff
@@ -307,6 +307,24 @@ class DatabaseManager {
         return undefined;
     };
 
+    public getAppDatabaseAndOperator = () => {
```

Comment:
Something like:


```
/**
 * Helper method that returns the database and operator;
 * use within a try/catch block
 *
 * @returns { operator: ServerDataOperator, database: Database }
 * @throws Error 
 *
```

Reply:
That would just be a function documentation, isn't it ?

Reply:
Yes that's the one! Couldn't remember what it was called exactly 😅

Reply:
@shazm added